include/nbl/builtin/hlsl/math/constants.hlsl

```diff
@@ -0,0 +1,15 @@
+
+// Copyright (C) 2018-2022 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _NBL_BUILTIN_HLSL_MATH_CONSTANTS_INCLUDED_
+#define _NBL_BUILTIN_HLSL_MATH_CONSTANTS_INCLUDED_
+
+#include <nbl/builtin/hlsl/limits/numeric.hlsl>
+
+#define PI 3.14159265359
+#define RECIPROCAL_PI 0.318309886183
+#define SQRT_RECIPROCAL_PI 0.56418958354
```

Comment:
I'd want to wrap these up the same way C++20 does

so lets make a namespace and a struct with `static const` default initialized members